collagraph/component.py

```diff
@@ -43,3 +54,7 @@ def before_unmount(self):
     @abstractmethod
     def render():  # pragma: no cover
         pass
+
+    def __del__(self):
+        # Make sure to clean-up _element.
+        self._element = None
```

Comment:
I'm just thinking, `__del__` is called when this instance is being garbage collected. That would mean the `self._element` is also being removed. So is this method really doing anything?

Reply:
Yeah, you're right!